src/main/java/core/mate/academy/service/MachineProducer.java

```diff
@@ -1,11 +1,13 @@
 package core.mate.academy.service;
 
+import core.mate.academy.model.Machine;
 import java.util.List;
 
 public interface MachineProducer {
```

Comment:
**interface is not parametrized**
![image](https://user-images.githubusercontent.com/20326858/201078890-72044c72-a635-4dd4-9d64-e2f557635657.png)